<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>680</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>680</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Poli Weather</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(51, 73, 95);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0" colspan="2">
     <widget class="QFrame" name="centralFrame">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 170, 255, 30);</string>
      </property>
      <layout class="QGridLayout" name="layoutInner">
       <item row="2" column="0">
        <widget class="QLabel" name="weatherIconMain">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgba(0,0,0,0);</string>
         </property>
         <property name="text">
          <string>text</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QLabel" name="fullPlaceName">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 75 15pt &quot;DejaVu Sans&quot;;
background-color:rgba(0,0,0,0);</string>
         </property>
         <property name="text">
          <string>No Data Loaded</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="4">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="Line" name="line">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="4">
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="4">
        <widget class="QScrollArea" name="scrollArea">
         <property name="minimumSize">
          <size>
           <width>360</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>360</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">/*QScrollBar:handle:vertical {
background: rgb(170, 170, 255);
}
QScrollBar:up-arrow:vertical, QScrollBar::down-arrow:vertical {
background:rgb(170, 170, 255);
}*/</string>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>358</width>
            <height>278</height>
           </rect>
          </property>
          <widget class="QListWidget" name="listWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>1</y>
             <width>361</width>
             <height>291</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="Description">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font: 75 13pt &quot;DejaVu Sans&quot;;
background-color:rgba(0,0,0,0);</string>
           </property>
           <property name="text">
            <string>text</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Wind">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font: 75 13pt &quot;DejaVu Sans&quot;;
background-color:rgba(0,0,0,0);</string>
           </property>
           <property name="text">
            <string>text</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Temperature">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font: 75 13pt &quot;DejaVu Sans&quot;;
background-color:rgba(0,0,0,0);</string>
           </property>
           <property name="text">
            <string>text</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Umidity">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
font: 75 13pt &quot;DejaVu Sans&quot;;
background-color:rgba(0,0,0,0);</string>
           </property>
           <property name="text">
            <string>text</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdit">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 255);</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QFrame" name="bottomFrame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QPushButton" name="filters">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/images/images/language.png</normaloff>:/images/images/language.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="emptyLabel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="options">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/images/images/settings.png</normaloff>:/images/images/settings.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Abyssinica SIL&quot;;
color: rgb(255, 255, 255);
background-color: rgb(20, 116, 208);</string>
      </property>
      <property name="text">
       <string>Check</string>
      </property>
      <property name="shortcut">
       <string>Return</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QFrame" name="upperFrame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>90</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="logoLabel2">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>90</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>90</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="logoLabel1">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>90</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>90</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(:/images/images/logo-scritta.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
